recycle-node.rb

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env ruby
+
+
+# Usage:
+#   export AWS_PROFILE=moj-cp
+#   Edit "K8S_CLUSTER_NAME" to specify the cluster to recycle old node. 
+#   Set kubectl config to use-context #K8S_CLUSTER_NAME
+#   ./create-cluster.rb
+#
+#  Note: To run this on a test cluster, update function `get_worker_instance_group_size` as shown below, 
+#  to return number of worker nodes configured as minSize in kops/test-cluster.yaml file for the test cluster.
+# 
+#  def get_worker_instance_group_size
+#     return 3 
+
+require 'json'
+require "yaml"
+require "net/http"
+
+K8S_CLUSTER_NAME = "recycle-node.cloud-platform.service.justice.gov.uk"
+AWS_REGION = "eu-west-2"
+KOPS_CONFIG_URL = "https://raw.githubusercontent.com/ministryofjustice/cloud-platform-infrastructure/master/kops/live-1.yaml"
+
+def main
+  number_of_workers = count_worker_nodes
```

Comment:
line 25 doesn't do anything useful, since the logic testing the number is in `correct_number_of_workers_running?`

I'd also replace `worker_instance_group_size` with a call to `get_ worker_instance_group_size` on line 29, so you can remove line 26, too